testsuite/tests/shadow_include/shadow_all.ml

```diff
@@ -163,20 +163,18 @@ module type Module_type = sig
 end
 ;;
```

Comment:
Nitpick: the convention in these tests is apparently that examples that are known to fail should be called `Foo_fail`, so here I would expect `Module_type_fail`. The name is already used below, but that is okay.